tests/blas_like/Multiply.cpp

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
I am only covering the routines I added for now. I have not yet added any Distributed multiply's. WHen I add then they will get tests, and i'll try at that point to cover more routines.